tests/test_runtime.py

```diff
@@ -173,3 +188,8 @@ def test_error_state(self):
                 self.runtime.start_and_wait_completed(sync=True)
 
         assert self.runtime.state == RuntimeStates.error, self.runtime.state
+
+    @pytest.mark.asyncio
+    async def test_runtime_cancelled(self):
+        """Test runtime cancelled."""
+        pytest.skip("only for async runtime")
```

Comment:
this is... just very odd. Can this be removed or does it serve some esoteric purpose? If in fact the latter please explicate.

Reply:
updated properly

Reply:
I don't understand this test though. What does it do? What is its purpose?